policyengine_us/parameters/gov/states/nc/ncdhhs/scca/childcare_market_rates.yaml

```diff
@@ -0,0 +1,913 @@
+description: North Carolina set the following Subsidized Child Care Market Rates for Child Care Centers.
+metadata:
+  unit: currency-USD
+  period: month
+  label: North Carolina Subsidized Child Care Assistance (SCCA) program market rates brake down by county and child age
+  breakdown:
+    - county
+    - nc_scca_age_group
+  reference:
+    - title: Subsidized Child Care Market Rates for Child Care Centers
+      href: https://ncchildcare.ncdhhs.gov/Portals/0/documents/pdf/M/Market_Rates_Centers_Eff_10-1.pdf?ver=9w52alSPhmrmo0N9gGVMEw%3d%3d
+    - title: Estimated Savings NC Subsidized Child Care Assistance Program
+      href: https://docs.google.com/spreadsheets/d/1y7p8qkiOrMAM42rtSwT_ZXeA5tzew4edNkrTXACxf4M/edit?gid=1339413807#gid=1339413807 
```

Comment:
is there a publicly available version of this?

Reply:
Adding a comment to explain how these rates are determined.